packages/token-lab/src/components/EditableFontFamilyToken/EditableFontFamilyToken.tsx

```diff
@@ -0,0 +1,160 @@
+import { move } from '@dnd-kit/helpers';
+import { DragDropProvider } from '@dnd-kit/react';
+import { useSortable } from '@dnd-kit/react/sortable';
+import { CaretSort, Cross } from '@terrazzo/icons';
+import { Button, SubtleInput } from '@terrazzo/tiles';
+import type { FontFamilyTokenNormalized } from '@terrazzo/token-tools';
+import { useReducer } from 'react';
+import c from './EditableFontFamilyToken.module.css';
+
+type SortableProps = {
+  id: string | number;
+  index: number;
+  children?: React.ReactNode;
+};
+
+function Sortable({ id, index, children }: SortableProps) {
+  const { ref } = useSortable({ id, index });
+
+  return (
+    <li ref={ref} className={c.item}>
+      <CaretSort />
+      {children}
+    </li>
+  );
+}
+
+// function SortableList() {
+//   const items = [1, 2, 3, 4];
+
+//   return (
+//     <ul className={c.list}>
+//       {items.map((id, index) => (
+//         <Sortable key={id} id={id} index={index} />
+//       ))}
+//     </ul>
+//   );
+// }
```

Comment:
TODO: pull sortable out into a reusable component

Reply:
TODO: adjust behavior of sorting so that items don't move until dropped. This is probably deferrable since it's a UX issue. There are other higher priority concerns.

Here's a demo of the desirable behavior though: https://shadcn-drag-and-drop.vercel.app/